core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -650,6 +646,23 @@ object ConsumerGroupCommand extends Logging {
       }
     }
 
+    private def checkOffsetRange(topicPartition: TopicPartition, shiftedOffset: Long) = {
+      val newOffset = getLogEndOffset(topicPartition) match {
```

Comment:
nit: we don't really need the `newOffest` variable. We can return from the match directly.